templates/ros2_control/controller/dummy_package_namespace/dummy_chainable_controller.hpp

```diff
@@ -35,10 +35,13 @@
 namespace dummy_package_namespace
 {
 // name constants for state interfaces
-static constexpr size_t STATE_MY_ITFS = 0;
+static constexpr size_t NR_STATE_ITFS = 0;
 
 // name constants for command interfaces
-static constexpr size_t CMD_MY_ITFS = 0;
+static constexpr size_t NR_CMD_ITFS = 0;
+
+// name constants for reference interfaces
+static constexpr size_t NR_REF_ITFS = 0;
```

Comment:
Now we are using this variables, should they really be 0?

Reply:
you are right i have the changes locally done did not push them yet. Will do it